engine/definitions.go

```diff
@@ -86,39 +82,24 @@ const (
 	satoshisField        = "satoshis"
 	spendingTxIDField    = "spending_tx_id"
 	statusField          = "status"
-	syncStatusField      = "sync_status"
 	typeField            = "type"
 	xPubIDField          = "xpub_id"
 	xPubMetadataField    = "xpub_metadata"
-	blockHeightField     = "block_height"
-	blockHashField       = "block_hash"
-	merkleProofField     = "merkle_proof"
-	bumpField            = "bump"
-	fullNameField        = "full_name"
 	paymailField         = "paymail"
 	contactStatusField   = "status"
 
 	// Universal statuses
-	statusCanceled   = "canceled"
-	statusComplete   = "complete"
-	statusDraft      = "draft"
-	statusError      = "error"
-	statusExpired    = "expired"
-	statusPending    = "pending"
-	statusProcessing = "processing"
-	statusReady      = "ready"
-	statusSkipped    = "skipped"
+	statusCanceled = "canceled"
+	statusComplete = "complete"
```

Comment:
Minor: `completed` would be better
```suggestion
	statusCompleted = "completed"
```

Reply:
That's some constant - defined long ago. It potentially will be removed when we get rid of drafts. 
So - thanks for pointing that out - but it's out of scope of this task 🙈 Won't change 😄